packages/manager/src/features/CloudPulse/Alerts/ContextualView/AlertInformationActionTable.tsx

```diff
@@ -120,6 +120,18 @@ export const AlertInformationActionTable = (
     setIsDialogOpen(true);
     setSelectedAlert(alert);
   };
+
+  let isActive = false;
```

Comment:
Nit:  Better naming for this variable. 
Something like the suggested change, because it is used to check if entity is enabled or not.
```suggestion
  let isEnabled = false;
```

Reply:
thanks for pointing out, I've missed here